<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddRemoveListView</class>
 <widget class="QWidget" name="AddRemoveListView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLineEdit" name="leItemEdit"/>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="btnAdd">
     <property name="text">
      <string>Add</string>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/images/plus.png</normaloff>:/images/plus.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QListView" name="ivItemList">
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1" alignment="Qt::AlignTop">
    <widget class="QPushButton" name="btnRemove">
     <property name="text">
      <string>Remove</string>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/images/minus.png</normaloff>:/images/minus.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
